packages/core/src/shader/utils/Preprocessor.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * Created by hazed on 30.01.2020.
+ */
```

Comment:
Is it needed?

Reply:
Agreed, please remove. We use git history to determine authorship and time stamping.

Reply:
ok, its auto-generate by WebStorm